fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java

```diff
@@ -781,6 +782,9 @@ private void handleQueryWithRetry(TUniqueId queryId) throws Exception {
                 handleQueryStmt();
                 break;
             } catch (RpcException | UserException e) {
+                if (Config.isCloudMode() && e.getMessage().contains("E-230")) {
```

Comment:
hard code string is not a good idea for maintainance

Reply:
fixed